package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-gridstack",
-  "version": "1.3.0",
+  "version": "1.4.0",
```

Comment:
Isn't this a breaking change? `touch-punch` went from opt-out to opt-in, and developers using it now need to update to include it.

Reply:
@tacocat you are correct. The plan is to bump this to 2.x.